src/super_gradients/training/metrics/metric_utils.py

```diff
@@ -39,6 +39,40 @@ def calc_batch_prediction_detection_metrics_per_class(metrics, dataset_interface
     return results_tuple
 
 
+def calc_batch_prediction_detection_metrics_per_classV2(matching, dataset_interface, iou_thres, silent_mode,
```

Comment:
Please add docs.

Reply:
Same as previous comment, this function was removed because outdated and never used: f5182088bd85175c86d345622c1242a924c989e7